### PR TITLE
{lib}[gompi/2023a] mpifileutils v0.11.1, attr v2.5.1, dtcmp v1.1.4, ...

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.16
See https://gist.github.com/SebastianAchilles/fd465b6352f10d92a4f4790e26cd98f7 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
zen2-ubuntu-2304 - Linux Ubuntu 23.04 (Lunar Lobster), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.2
See https://gist.github.com/SebastianAchilles/5ce8e09358c7bdf37923ae2b416608a3 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.11.1-gompi-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ('lwgrp', '1.0.5'),
+    ('dtcmp', '1.1.4'),
+    ('libarchive', '3.6.2'),
+    ('libcircle', '0.3'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dsync', 'include/mfu.h', 'lib/libmfu.%s' % SHLIB_EXT],
+    'dirs': []
+}
```
Comment (smoors) [inline easybuild/easyconfigs/m/mpifileutils/mpifileutils-0.11.1-gompi-2023a.eb:35]: would it be worth it to add `sanity_check_commands` for `dsync` here?

File: easybuild/easyconfigs/a/attr/attr-2.5.1-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+builddependencies = [('binutils', '2.40')]
+
+sanity_check_paths = {
+    'files': ['bin/attr', 'bin/getfattr', 'bin/setfattr',
+              'include/%(name)s/attributes.h', 'include/%(name)s/error_context.h',
+              'include/%(name)s/libattr.h', 'lib/libattr.a',
+              'lib/libattr.%s' % SHLIB_EXT],
+    'dirs': ['share'],
+}
```
[the remark this answers is not in the document]
Comment (SebastianAchilles) [reply easybuild/easyconfigs/a/attr/attr-2.5.1-GCCcore-12.3.0.eb:24]: Thanks for the suggestion :+1: Implemented in https://github.com/easybuilders/easybuild-easyconfigs/pull/18950/commits/2ec6b5a07eda735589547f3ed197b9702b751a53

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS='--module-only'

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--module-only"

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**